### ISSUE TITLE
inspector should support --inspect-brk 

### ISSUE THREAD

**Issue by ry** [denoland/deno#4503]

**Comment by kevinkassimo** [comment]
I feel this is probably going to be quite tricky per some experiments I did (TBH I'm not familiar with the Inspector API so I might make some mistakes):

It seems that first we might need to make `DenoInspector` spawn in a different Tokio runtime from the `Worker`, and before `Worker` is ever evaluated. Current situation only polls the `DenoInspector` in `Worker::poll`, which makes it not possible to break before first run-to-the-end is completed.

Also, I've tried to do `session.schedule_pause_on_next_statement(...)` and does not seem to work even if the above is done. Maybe this is due to myself invoking this without any JS code of relevant context running of top the Isolate, thus there is no next statement really to break for. Node does not seem to have this problem due to its module instantiation/evaluation called from JS through a binding to `schedulePauseOnNextStatement()`:
https://github.com/nodejs/node/blob/517b0470c7e3147606425604dd9c8429cf0f4de4/lib/internal/modules/esm/module_job.js#L89-L90
